docker/Dockerfile-api

```diff
@@ -16,18 +16,19 @@ RUN dnf -y install \
     mod_ssl \
     python3-mod_wsgi \
     openssl-devel \
-    python3-devel \
-    python3-pip \
-    python3-wheel \
+    python38-devel \
+    python38-pip \
+    python38-wheel \
     && dnf update -y \
     && dnf clean all
+RUN update-alternatives --set python3 $(which python3.8)
 COPY . .
 COPY ./docker/iib-httpd.conf /etc/httpd/conf/httpd.conf
 
-# default python3-pip version for rhel8 python3.6 is 9.0.3 and it can't be updated by dnf
+# default python3-pip version for rhel8 python3.8 is 19.3.1 and it can't be updated by dnf
 # we have to update it by pip to version above 21.0.0
 RUN pip3 install --upgrade pip
```

Comment:
We should consider removing this. The latest version installed by dnf might sufficient now that it's using python 3.8. Same on `Dockerfile-workers`.

(Apologies for the late review)

Reply:
Excellent point!

We can discuss as a team and I can implement in next PR